packages/xo-web/src/xo-app/pool/tab-patches.js

```diff
@@ -187,11 +197,12 @@ export default class TabPatches extends Component {
               {ROLLING_POOL_UPDATES_AVAILABLE && (
                 <TabButton
                   btnStyle='primary'
-                  disabled={isEmpty(missingPatches)}
+                  disabled={isEmpty(missingPatches) || isSingleHost}
                   handler={rollingPoolUpdate}
                   handlerParam={pool.id}
                   icon='pool-rolling-update'
                   labelId='rollingPoolUpdate'
+                  tooltip={isSingleHost ? _('isSingleHost') : null}
```

Comment:
We usually use `undefined`.
```suggestion
                  tooltip={isSingleHost ? _('isSingleHost') : undefined}
```